Create style-guide.md (#184)
w3c/wai-website-theme: _style-guide/about.md

---
title: "About this Style Guide"
title_sentence: "About this style guide"
title_html: "About this Style Guide"
nav_title: "About"
lang: en
order: 0
---

The purpose of the guide is to support readability, consistency, and easier translations.

> This is a living document. It aims to follow the recommendations in this style guide.

Reviewer: ```suggestion
This is a living document. Substantive changes will be listed in the changelog.
```

> **Important:**  These guidelines are written for the **US English language**. Translators may adapt them to apply regional conventions. 

Reviewer: ```suggestion
This style guide is for US English. Translators can use conventions in their language.
```

rationale: It's not **important**. We don't want "regional" because we want global in almost all cases.

@remibetin thoughts?

Reviewer 2: +1 to removing "Important". 

I was not too bothered by "regional", but the updated wording also works for me.

As a data point, we use in our translation review guidance: "the translation uses respectful disability terminology in the target language and region".

[Minor] Suggestion: "Translators can use conventions in the target language and region."
